src/requests.ts

```diff
@@ -276,13 +290,20 @@ function makeRequest(
   method: string,
   item: OpenAPIV3.OperationObject,
   $refs: SwaggerParser.$Refs,
-  options: CLIOptions
+  options: CLIOptions,
+  pathParams?: OpenAPIV3.PathItemObject["parameters"]
 ) {
   const pathTemplateExpression = patternToPath(
     pattern,
     options.baseUrl,
     options.replacer
   );
+
+  const parameters = combineUniqueParams(
+    pathParams || [],
+    item.parameters || []
+  );
+  item.parameters = parameters;
```

Comment:
Same comments as above. Also this looks very similar to the other file's logic, can it be moved to `common.ts` and used in both places?